### DISCUSSION TITLE
Feedback for Creating a Console Project

### DISCUSSION THREAD

**Discussion by jcollard** [CaptainCoderOrg/IntroToCSharpSite#91] [Page Feedback]
Feedback for "Creating a Console Project": https://csharp-develop.captaincoder.org/lessons/dotnet/creating-a-console-project

**Comment by cloudthepain** [comment]
This might be a down the line thing but it might be a good idea to create some of our own videos for these links. That way we can update them and make sure that they are dependable. It would also be cool to have in the future if we wanted to have visual examples of certain activities!

**Reply by jcollard** [reply]
Absolutely! I have this YouTube channel: https://www.youtube.com/CaptainCoder

Ideally, we would have videos that go through a lot of the lessons / activities in the book. If you wanted to record a video walking through the lesson, that would be awesome!

**Reply by cloudthepain** [reply]
Perfect! That's something I might have to look into later down the line.

**Comment by cloudthepain** [comment]
Another thing that might be worth considering on pages is implementing a "Last Updated" on the pages to ensure that the pages are up to date and show the users that the page are still accurate and current.
This might be more important for features that might frequently change like .Net stuff.

**Reply by jcollard** [reply]
This is a great idea! I've added it to the project board: #92

**Comment by cloudthepain** [comment]
In this section did you want them to switch to bash? If you did I would add it after the "verify terminal is open" section.

**Reply by jcollard** [reply]
No. They should be able to keep using whichever terminal is the default on their system. Someday it would be great to add in `git` and have them install `Git Bash` on Windows / learn the command line tools for that. But it is currently out of scope for this part of the book.

**Reply by cloudthepain** [reply]
Understood.

**Comment by jcollard** [comment]
We should have them create a folder specifically for their work.
